nanome/_internal/_plugin.py

```diff
@@ -83,6 +84,8 @@ def __parse_args(self):
                     sys.exit(1)
                 split = sys.argv[i + 1].split(",")
                 self.__to_ignore.extend(split)
+            elif sys.argv[i] == "-nl":
+                self.__no_logs = True
```

Comment:
Logs will still show up in the terminal right? I would expect to see the output as it happens, and just not save it to a file.
Otherwise `__no_logs` is misleading

Reply:
You're right, __no_logs is misleading